src/common/hooks/useExtractTextFromHTML.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+export function useExtractTextFromHTML() {
```

Comment:
Is there a reason why this is marked as a hook?

Seems like more work and more lines of code if this is marked as hook, but it has no access to React's lifecycle nor state.